ppr-ui/src/components/parties/Debtors.vue

```diff
@@ -70,14 +70,18 @@
               :key="row.item.id"
               class="debtor-row"
             >
-              <td class="list-item__title title-text">
+              <td class="list-item__title title-text pl-7">
```

Comment:
please change this from pl-7 (28 px) to padding-left 30px so it matches the spec for the other summaries

Reply:
done